sds_data_manager/stacks/processing_stack.py

```diff
@@ -28,13 +28,14 @@ def __init__(
         processing_step_name: str,
         lambda_code_directory: str or Path,
         data_bucket: s3.Bucket,
-        instrument_target: str,
-        instrument_sources: str,
+        instrument: str,
+        instrument_dependents: dict,
+        dependents: str,
```

Comment:
`dependents` is a confusing name for this IMO. maybe `dependency_filepath` is more clear?